modular_skyrat/modules/decay_subsystem/code/decaySS.dm

```diff
@@ -0,0 +1,86 @@
+/*
+This is the decay subsystem that is run once at startup.
+These procs are incredibly expensive and should only really be run once. That's why the only run once.
+*/
+
+
+#define WALL_RUST_PERCENT_CHANCE 15
```

Comment:
if you make rust something that happens a lot you might want to consider a way to remove the rust easily too, tho that's out of the PRs scope, just a note

Reply:
Easiest way is tooling them

Reply:
doesnt work well with reinforced walls
i just wanna clean 😢

Reply:
Eee there needs to be a rust component then